nautobot/cloud/models.py

```diff
@@ -215,3 +221,17 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+@extras_features("graphql")
```

Comment:
Should this support custom_validators and webhooks at least as well?

Reply:
Looking at the new model checklist, I think the recommended minimum `extras_features` would all work here: `custom_links, custom_validators, export_templates, graphql, and webhooks`

Reply:
Actually, not webhooks since those require the model to be change logged.

Reply:
Hmm. *Should* this be a ChangeLoggedModel?

Reply:
Possibly. None of our similar through table models in IPAM are so I'd say we should track that in a follow-up to fix all of them at once?